exporter/logzioexporter/config_test.go

```diff
@@ -0,0 +1,34 @@
+package logzioexporter
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/config/configtest"
+	"path"
+	"testing"
```

Comment:
adopting import order conventions would be appreciated:
stdlib
external
collector-contrib local